datafusion/physical-expr/src/expressions/in_list.rs

```diff
@@ -1543,237 +932,12 @@ mod tests {
         Ok(())
     }
 
-    #[test]
```

Comment:
We no longer need these duplicated tests as there isn't a separate execution path based on the length of the InList